std/stdio.d

```diff
@@ -999,6 +1005,12 @@ Throws: `Exception` if the file is not opened or if the OS call fails.
             import core.sys.windows.winbase : FlushFileBuffers;
             wenforce(FlushFileBuffers(windowsHandle), "FlushFileBuffers failed");
         }
+        version (Darwin)
+        {
+            import core.sys.darwin.fcntl : fcntl, F_FULLFSYNC;
```

Comment:
std/stdio.d(1010): Error: module `fcntl` is in file 'core/sys/darwin/fcntl.d' which cannot be read

Reply:
https://github.com/dlang/druntime/pull/3372 I see